template_ws/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ros:humble
```

Comment:
I think we should change the base image to `osrf/ros:humble-desktop-full` for the ease of future development.